sentinel/src/androidTest/AndroidManifest.xml

```diff
@@ -6,12 +6,24 @@
     <uses-sdk tools:overrideLibrary="androidx.security"/>
 
     <application
-        android:name=".ui.SentinelTestApplication"
+        android:name="com.infinum.sentinel.ui.SentinelTestApplication"
         android:label="@string/sentinel_name">
         <activity
-            android:name=".ui.main.SentinelActivity"
+            android:name="com.infinum.sentinel.ui.main.SentinelActivity"
             android:label="@string/sentinel_name"
             android:theme="@style/Sentinel.Theme" />
+
+        <provider
+            android:name="androidx.startup.InitializationProvider"
+            android:authorities="${applicationId}.androidx-startup"
+            android:exported="false"
+            tools:node="merge">
+
+            <meta-data
+                android:name="androidx.work.WorkManagerInitializer"
+                android:value="androidx.startup"
+                tools:node="remove" />
+        </provider>
```

Comment:
I can't remember if there were any conflicts, but it is possible it is from that. I will re-check this.

Reply:
Okay, I see now. This is from initial commit to this branch by Neven when we were figuring out what is wrong with the tests. I will check if this can be removed.